### PR TITLE
Code clean ups

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by solutionDrive-Alt** [comment]
you can fix your build by running 

`bin/easy-coding-standard check src/ spec/ --fix`

this will fix the one issue with the coding-standards.

File: src/Controller/ImportDataController.php [hunk 1]
```diff
@@ -23,13 +23,23 @@
     /** @var Session */
     private $session;
 
+    /**
+     * @param ServiceRegistry $registry
+     * @param UrlGeneratorInterface $router
+     * @param Session $session
+     */
     public function __construct(ServiceRegistry $registry, UrlGeneratorInterface $router, Session $session)
```
Comment (bitbager) [inline src/Controller/ImportDataController.php]: Be more DIish, change parameters type to `ServiceRegistryInterface` and `SessionInterface` :)

File: src/Controller/ImportDataController.php [hunk 2]
```diff
@@ -40,6 +50,11 @@ public function importAction(Request $request): RedirectResponse
         return new RedirectResponse($this->router->generate('sylius_admin_' . $importer . '_index'));
     }
 
+    /**
+     * @param Request $request
+     * @param $importer
+     * @param $format
+     */
     private function importData(Request $request, $importer, $format): void
```
Comment (bitbager) [inline src/Controller/ImportDataController.php]: These parameters should be strict typed.

File: src/DependencyInjection/Configuration.php [hunk 3]
```diff
@@ -9,10 +9,13 @@
 
 class Configuration implements ConfigurationInterface
```
Comment (bitbager) [inline src/DependencyInjection/Configuration.php]: Make it final

File: src/DependencyInjection/SyliusImportExportExtension.php [hunk 4]
```diff
@@ -11,10 +11,13 @@
 
 class SyliusImportExportExtension extends Extension
```
Comment (bitbager) [inline src/DependencyInjection/SyliusImportExportExtension.php]: Final again

File: src/SyliusImportExportPlugin.php [hunk 5]
```diff
@@ -13,6 +13,9 @@
 {
     use SyliusPluginTrait;
 
+    /**
+     * {@inheritdoc}
+     */
     public function build(ContainerBuilder $container)
```
Comment (bitbager) [inline src/SyliusImportExportPlugin.php]: Add the `:void` return type.

File: src/DependencyInjection/SyliusImportExportExtension.php [hunk 6]
```diff
@@ -11,10 +11,13 @@
...
 {
+    /**
+     * {@inheritdoc}
+     */
     public function load(array $configs, ContainerBuilder $container)
```
Comment (bitbager) [inline src/DependencyInjection/SyliusImportExportExtension.php]: Don't forget about the return type ;)

File: src/Command/ImportDataCommand.php [hunk 7]
```diff
@@ -76,6 +76,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $output->writeln($message);
+
+        return 0;
     }
 
     private function listImporters(InputInterface $input, OutputInterface $output, ServiceRegistry $registry): void
```
Comment (bitbager) [inline src/Command/ImportDataCommand.php]: Hey, ServiceRegistryInterface* :)

**Pull request merged**